website/docs/guides/migration/versions/02-upgrading-to-v1.5.md

```diff
@@ -56,7 +56,50 @@ models:
     tests: [] # todo! add tests later
     config: ...
 ```
-Some options that could previously be specified before a sub-command can now only be specified afterward. For example, `dbt --profiles-dir . run` isn't valid anymore, and instead, you need to use `dbt run --profiles-dir .`
+
+Some options that could previously be specified _after_ a subcommand can now only be specified _before_. This includes the inverse of the command, `--write-json` and `--no-write-json`, for example.  The list of affected commands are:
+
+<details>
+<summary>List of affected commands</summary>
+
+```bash
+--cache-selected-only | --no-cache-selected-only
+--debug, -d | --no-debug
+--deprecated-print | --deprecated-no-print
+--enable-legacy-logger | --no-enable-legacy-logger
+--fail-fast, -x | --no-fail-fast
+--log-cache-events | --no-log-cache-events
+--log-format
+--log-format-file
+--log-level
+--log-level-file
+--log-path
+--macro-debugging | --no-macro-debugging
+--partial-parse | --no-partial-parse
+--partial-parse-file-path
+--populate-cache | --no-populate-cache
+--print | --no-print
+--printer-width
+--quiet, -q | --no-quiet
+--record-timing-info, -r
+--send-anonymous-usage-stats | --no-send-anonymous-usage-stats
+--single-threaded | --no-single-threaded
+--static-parser | --no-static-parser
+--use-colors | --no-use-colors
+--use-colors-file | --no-use-colors-file
+--use-experimental-parser | --no-use-experimental-parser
+--version, -V, -v
+--version-check | --no-version-check
+--warn-error
+--warn-error-options
+--write-json | --no-write-json
+
+```
+
+</details>
+
+
+Additionally, some options that could be previously specified _before_ a subcommand can now only be specified _after_. Any command _not_ on the above list must appear _after_ the subcommand from v1.5 on. For example, `--profiles-dir`.
```

Comment:
```suggestion
Additionally, some options that could be previously specified _before_ a subcommand can now only be specified _after_. Any command _not_ within the above list must appear _after_ the subcommand from v1.5 on. For example, `--profiles-dir`.
```

Reply:
```suggestion
Additionally, some options that could be previously specified _before_ a subcommand can now only be specified _after_. Any command _not_ in the above list must appear _after_ the subcommand from v1.5 on. For example, `--profiles-dir`.
```

Reply:
good catch, @dbeatty10 ! went with "in" -- slightly simpler